lib/modules/configure.go

```diff
@@ -152,8 +152,14 @@ func getBlockConfigInstance(m map[string]interface{}) (*BlockConfig, error) {
 		err := yaml.Unmarshal(yamlStr, &c)
 		b := BlockConfig(c)
 		return &b, err
+	case "command":
+		c := Command{}
+		err := yaml.Unmarshal(yamlStr, &c)
+		b := BlockConfig(c)
+		return &b, err
 	}
-	return nil, fmt.Errorf("error: type %s not valid", t)
+
+	return nil, fmt.Errorf("type %s not valid", t)
```

Comment:
The error messages already had an "error:" string so it was getting repeated twice when errors were generated.